src/abr/puffer_ttp.cc

```diff
@@ -35,11 +36,33 @@ PufferTTP::PufferTTP(const WebSocketClient & client,
       ttp_input_dim_ = 17;
       no_tcp_info_ = true;
     }
+    if (abr_config["blur_params"]) {
+      mean_val_ = abr_config["blur_params"]["mean_val"].as<double>();
+      std_val_ = abr_config["blur_params"]["std_val"].as<double>();
+      kernel_size_ = abr_config["blur_params"]["kernel_size"].as<int>();
+      /* In our current blur cases, kernel size has
```

Comment:
Nit: each line can have 80 chars at most, so you don't need to break the sentence too often